src/deployment/android.md

```diff
@@ -301,16 +301,16 @@ A [multidex keep file][multidex-keep] must be specified to include:
 io/flutter/embedding/engine/loader/FlutterLoader.class
 io/flutter/util/PathUtils.class
 ```
+
 Also, include any other classes used in app startup.
 For more detailed guidance on adding multidex support manually,
 check out the official [Android documentation][multidex-docs].
 
 ## Reviewing the app manifest
 
-Review the default [App Manifest][manifest] file,
-`AndroidManifest.xml`,
-located in `[project]/android/app/src/main` and
-verify that the values are correct, especially the following:
+Review the default [App Manifest][manifest] file, `AndroidManifest.xml`.
+This file is located in `[project]/android/app/src/main`.
+Verify that the following values:
```

Comment:
```suggestion
Verify that the following values are correct:
```

Reply:
That would be [redundant](https://www.merriam-webster.com/dictionary/verify).

Reply:
But "that" isn't explained in "Verify that the following values:". Perhaps "Verify the following values"?